src/main/java/duke/DoneTask.java

```diff
@@ -0,0 +1,34 @@
+package duke;
+
+/**
+ * DoneTask command that contains index of Task we want to mark done
+ *
+ * @author wj331
+ */
+public class DoneTask extends Command {
+    private int taskDone;
```

Comment:
```suggestion
    private int taskIndex;
```
taskDone as a variable name might imply a boolean (task done or not). A clearer name could be taskIndexToMarkDone or simply taskIndex.